demos/twa-webview-fallback/src/main/AndroidManifest.xml

```diff
@@ -41,6 +41,9 @@
             <meta-data android:name="android.support.customtabs.trusted.ADDITIONAL_TRUSTED_ORIGINS"
                 android:resource="@array/additional_trusted_origins" />
 
+            <meta-data android:name="android.support.customtabs.trusted.FALLBACK_STRATEGY"
+                android:value="webview" />
```

Comment:
is there a reason why we are preferring webviews over customtabs?

I'm not sure what the compat is like, but presumably customtabs should be preferred due to the matching API parity

Reply:
We're preferring CustomTabs over WebView. The demo shows how to change the default, so it has the meta-tag.

Reply:
TL;DR is that yes, compat is bad and developers shouldn't expect the WebView fallback to work seamlessly, but some users get frustrated when they get the Custom Tabs experience, so developers having been asking for a WebView fallback. We still recommend CustomTabs for compatibility. More browsers are working on Trusted Web Activity support, so the need for a fallback should decrease soon.

Reply:
How come we're using webview instead of WebView? (Or is it just case insensitive?)

Reply:
The value is defined inside LauncherActivity:

```
    private static final String FALLBACK_TYPE_WEBVIEW = "webview";
```